### ISSUE TITLE
Couldn't log learning rate

### ISSUE THREAD

**Issue by amirshamaei** [ashleve/lightning-hydra-template#618]
I addded the following callback config to plot learning rate;however, nothing showed up.
Could you please help me to show learning rate during training?
```javascript
learning_rate_logger:
  _target_: lightning.pytorch.callbacks.LearningRateLogger
  logging_interval: 'step'
```

**Comment by AzharSindhi** [comment]
lightning.pytorch.callbacks.LearningRateLogger is renamed to lightning.pytorch.callbacks.LearningRateMonitor according to [https://github.com/Lightning-AI/lightning/issues/6418](url)

**Comment by gsoykan** [comment]
For the records, here is the snippet, I add it to configs/callbacks/default.yaml
```yaml
learning_rate_logger:
  _target_: lightning.pytorch.callbacks.LearningRateMonitor
  logging_interval: 'step'
```
